### DISCUSSION TITLE
Are these codes equivalent!!

### DISCUSSION THREAD

**Discussion by Abhi-SimuTech** [ansys/pyaedt#3643] [General]
Hi,

What may be the differences (in output geometry) if I replace the UDM code section in (1) , with PyAEDT function in (2):

(1) 
![image](https://github.com/ansys/pyaedt/assets/145813888/7ec08388-9867-4935-8e3a-3da0bc10c440)

(2) 
![image](https://github.com/ansys/pyaedt/assets/145813888/ddf41a13-5976-4b4f-9776-e2c8a3b2fca9)

where, 
![image](https://github.com/ansys/pyaedt/assets/145813888/f2f64f12-bd2a-4b5c-8cd8-1f64c9848c8f)

**Comment by Abhi-SimuTech** [comment]
Submitted at the wrong place, so closing this discussion.

**Reply by MaxJPRey** [reply]
Hi @Abhi-SimuTech,
Based on your last comment, I will close this discussion.
Regards.
